.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: 'test'
+
+# yamllint disable-line rule:truthy
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  go-test:
+    name: go-test
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        go-version: ['1.22.0']
```

Comment:
Can you modify this to just use the go version specified in the go.mod?

Reply:
Updated 👍